### DISCUSSION TITLE
how to change track thickness in slider

### DISCUSSION THREAD

**Discussion by harthorst** [siteline/SwiftUI-Introspect#251] [Q&A] [answered]
First: many thanks for providing this library. 

I'd like to customize the Sliders track size. For UISlider i was able to create a subclass and override trackRect(). How can I achieve the same result in SwiftUI and Introspect?

**Comment by davdroman** [comment, marked as answer]
Subclassing is indeed not an option, so the only other way to accomplish this is by dipping down into [swizzling](https://pspdfkit.com/blog/2019/swizzling-in-swift/) territory.

**Comment by schwjustin** [comment]
@harthorst did you figure this out

**Comment by harthorst** [comment]
No. I simply created my own slider widget which was easier than I thought.
